### PR TITLE
Apicalls

### PR DESCRIPTION
**Pull request by sakshi006**
Fixed : Cart and Wishlist APIs
Added paths for when user is logged in and logged  out.

### CODE REVIEW

File: src/Components/CartCard/CartCard.jsx [hunk 1]
```diff
@@ -4,8 +4,9 @@ import { useCartContext ,useWishListContext} from "../../context";
 import { calcPercentage } from "../../Utils";
 
 const CartCard = ({ cartproduct }) => {
-  const { setCartProducts } = useCartContext();
-  const { wishListState, setWishListState } = useWishListContext();
+  const token = localStorage.getItem("token")
```
Comment (sashaboi) [inline src/Components/CartCard/CartCard.jsx:7]: here if token returns undefined, you could write login fore navigate logic if you havent used auth decorator.

File: src/Components/CartCard/CartCard.jsx [hunk 2]
```diff
@@ -19,23 +20,24 @@ const CartCard = ({ cartproduct }) => {
...
-            Discount :{" "}
+            Discount :
             {calcPercentage(cartproduct.price, cartproduct.originalPrice)}%
           </span>
           <div className="qty-monitor">
             <button
               className="btn qty-btn"
               onClick={() =>
-                setCartProducts({ type: "DECREMENT_QTY", payload: cartproduct })
+               {if(cartproduct.qty>1)
```
Comment (sashaboi) [inline src/Components/CartCard/CartCard.jsx:30]: here on an else you could fire the remove from cart api as well?
seems counterintuitive to see button , be able to click on button, but button doesnt do anything.

File: src/backend/db/products.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { v4 as uuid } from "uuid";
+// import { v4 as uuid } from "uuid";
```
Comment (sashaboi) [inline src/backend/db/products.js:1]: why no uuid, causing some error?

File: src/Pages/Authentication/Login.jsx [hunk 4]
```diff
@@ -1,37 +1,86 @@
...
-          ><Link to="/signup"class="new-acc"
-            >Create New Account <i class="fas fa-chevron-right"></i
-          ></Link>
-        </form>
-      </div>
-  )
-}
+          <span className="forgot-pass">Forgot your Password?</span>
+        </div>
+        <button onClick={addDummyUser} className="btn login-btn">
```
Comment (sashaboi) [inline src/Pages/Authentication/Login.jsx:74]: nice touch, easy for people.

**Review by sashaboi — APPROVED** [review]
Everything looks amazing! 

there is this _redirects file  which once added to the code, helps not throw 404 on redirects which is a netlify issue.
made a few suggestions, looks good to merge!

**Review by krgarima — APPROVED** [review]
Add _redirect file so that when user clicks on back icon, he is not shown a 'Page not found' error message. Otherwise, everything this look to merge.

File: src/backend/db/products.js (hunk 3 above)
Comment (sakshi006) [reply src/backend/db/products.js:1]: uuid for each product changes everytime if  it is reloaded, so instead i provided a hardcoded  ID

File: src/Components/CartCard/CartCard.jsx (hunk 2 above)
Comment (sakshi006) [reply src/Components/CartCard/CartCard.jsx:30]: i have created a seperate button for remove from cart

**Pull request merged**